src/main/java/edu/cornell/kfs/rass/batch/RassStep.java

```diff
@@ -29,10 +30,9 @@ public boolean execute(String jobName, Date jobRunDate) throws InterruptedExcept
             return true;
         }
         List<RassXmlFileParseResult> successfulResults = findSuccessfullyParsedFileResults(parseResults);
-        RassXmlProcessingResults processingResults = rassService.updateKFS(successfulResults);
+        Map<String, RassXmlFileProcessingResult> processingResults = rassService.updateKFS(successfulResults);
         RassBatchJobReport report = new RassBatchJobReport(parseResults, processingResults);
-        rassReportService.writeBatchJobReport(report);
-        sendReport(report);
```

Comment:
it doesn't look like this function is used in RassStep any longer, should it be removed?

Reply:
Good catch, I'll remove that method.